maven-plugins/enforcer-maven-plugin/pom.xml

```diff
@@ -147,7 +146,6 @@
                             <overWriteIfNewer>true</overWriteIfNewer>
                             <overWriteIfNewer>true</overWriteIfNewer>
                             <includeScope>runtime</includeScope>
-                            <excludeScope>test</excludeScope>
```

Comment:
I updated the version of this plugin and after that maven builds started to fail because of this line.
According to the [documentation](https://maven.apache.org/plugins/maven-dependency-plugin/copy-dependencies-mojo.html#excludescope) :

test exclude scope excludes all dependencies, then not really a legitimate option: it will fail, you probably meant to configure includeScope = compile

So I thought that this line is not needed here

Reply:
Ok. Not sure why this worked before then.